libflux/flux-core/src/semantic/convert.rs

```diff
@@ -1,8 +1,9 @@
 //! Various conversions from AST nodes to their associated
 //! types in the semantic graph.
 
-use std::{collections::BTreeMap, fmt, rc::Rc};
+use std::{collections::BTreeMap, fmt, sync::Arc};
```

Comment:
Curious why you changed to `Arc` here, do you expect that something mutable will be accessed concurrently by multiple requests?

Reply:
I think `Symbol` ended up inside an `anyhow::Error` so it had to be `Send` due to that.